src/authenticationservices.cs

```diff
@@ -623,6 +636,10 @@ interface ASAuthorizationSingleSignOnCredential : ASAuthorizationCredential {
 
 		[NullAllowed, Export ("authenticatedResponse", ArgumentSemantic.Copy)]
 		NSHttpUrlResponse AuthenticatedResponse { get; }
+
+		[Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("privateKeys")]
+		NSObject[] PrivateKeys { get; }
```

Comment:
```
+/*! @abstract Private SecKeys returned from the AuthenticationServices extension.
+ */
+@property (nonatomic, readonly) NSArray *privateKeys API_AVAILABLE(ios(15.0), macos(12.0)) API_UNAVAILABLE(tvos, watchos);
```

`SecKey` is a type - so it should be an array of them

Reply:
Hmm, so I see that SecKey is created in this manual class here src/Security/Certificate.cs. 
The namespace is Security, but when I add 'using Security' the compiler still says that it cannot find the reference.

Reply:
That might require some changes in the generator. It does not seems to allow `SecKey` at the moment.

Still it's important because `SecKey` is **not** a subclass of `NSObject`, so the binding are incorrect without that fix.

Reply:
Should I create an issue and link that in the binding?

Reply:
yes, comment the bindings and create an issue
also add it to the checklist for xcode13

Reply:
Clarification: leave a comment saying this should be SecKey or comment out this whole property and leave a note there about it?

Reply:
* Comment the property binding
* Add a link to the issue you created inside the comment
* Add a link to the issue to https://github.com/xamarin/xamarin-macios/issues/11883

Why ? 
* It's not usable as an `NSObject` (since `SecKey` is not a subclass of it)
* if we forget it (or lack time to update) then we're not creating a future breaking change

It's always easier to add (later) than to remove :) 
IOW never add something "new" that's not right / needs to be fixed

Reply:
Not an issue with the generator. SecKey was in a #if COREBUILD moving the #if to provide the minimum fixed the problem. I have pushed the fix.